docs/ref/databases.txt

```diff
@@ -474,6 +474,32 @@ like other MySQL options: either in a config file or with the entry
 ``'init_command': "SET sql_mode='STRICT_TRANS_TABLES'"`` in the
 :setting:`OPTIONS` part of your database configuration in :setting:`DATABASES`.
 
+.. _mysql-tx-isolation-level:
+
+Transaction isolation levels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.11
+
+When running concurrent loads, database transactions from different sessions
+(say, separate threads handling different requests) may interact with each
+other.  These interactions are affected by the respective sessions'
+`transaction isolation levels`_. You can set your connection's transaction
+isolation level via a ``'isolation_level'`` entry in the :setting:`OPTIONS`
+part of your database configuration in :setting:`DATABASES`. Valid values for
+this entry are the four standard isolation levels (``'read uncommitted'``,
+``'read committed'``, ``'repeatable read'`` and ``'serializable'``), or
+``None`` to explicitly use the server's configured default.
+
+.. _transaction isolation levels: https://dev.mysql.com/doc/refman/en/innodb-transaction-isolation-levels.html
+
+MySQL's default transaction isolation level, ``REPEATABLE READ``, implies some
+`surprising behaviors`_. The Django project recommends using the ``READ COMMITTED``
```

Comment:
I don't think just saying 'surprising behaviours' and linking to MySQL docs is valid, we could maybe have a sentence to explain it's "Repeatable read BUT NOT reading your writes", maybe even link to the ticket reporting the bug on the admin (though the docs don't often do that I know)